src/java.desktop/share/classes/java/awt/image/BandedSampleModel.java

```diff
@@ -636,6 +600,9 @@ public int[] getSamples(int x, int y, int w, int h, int b,
      *                  object
      * @param data      The DataBuffer containing the image data
      * @see #getDataElements(int, int, Object, DataBuffer)
+     * @throws NullPointerException if {@code data} is null.
```

Comment:
I guess we will get NPE if (@code obj] is also null...Shouldn't we specify that too?

Reply:
I updated this as you can see.

Reply:
> Can we specify the NPEs in the same line like what it is done in l675 to be consistent and also to be concise? `@throws NullPointerException if {@code obj} or {code data} is null.`

I think I've done this now.